Meta/check-style.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "$script_path/.." || exit 1
+
+# We simply check if the file starts with:
+# /*
+#  * Copyright
+LICENSE_HEADER_PATTERN=$'^/\*\n \* Copyright'
+MISSING_LICENSE_HEADER_ERRORS=()
+
+# We check that "#pragma once" is present
+PRAGMA_ONCE_PATTERN='#pragma once'
+MISSING_PRAGMA_ONCE_ERRORS=()
+
+# We make sure that there's a blank line before and after pragma once
+GOOD_PRAGMA_ONCE_PATTERN=$'(^|\S\n\n)#pragma once(\n.|$)'
```

Comment:
This should be `(^|\S\n\n)#pragma once(\n\n\S|$)`: it wasn't actually checking the presence of a blank line, it also allowed the abscence of `\n` at the EOF.  
Let's also replace `\S` with `\\S` just in case.
```suggestion
GOOD_PRAGMA_ONCE_PATTERN=$'(^|\S\n\n)#pragma once(|$)'
```